src/utils/tips.js

```diff
@@ -0,0 +1,48 @@
+import storage from '../storage.js';
+
+class Tips {
+  constructor() {
+    this.tips = storage.get('tips');
```

Comment:
do we need a whole storage for this? could we store individual storage keys for these?